test2code-admin/src/main/kotlin/com/epam/drill/plugins/test2code/storage/Session.kt

```diff
@@ -36,30 +38,43 @@ data class AgentKey(
 internal class StoredSession(
     @Id val id: String,
     val agentKey: AgentKey,
-    val scopeId: String,
-    val data: FinishedSession,
+    @Id val sessionHolderId: String,
+    val testType: String,
+    val testName: String,
+    val tests: Set<TestOverview>,
+    var probes: List<ExecClassData>,
 )
 
 internal suspend fun StoreClient.loadSessions(
-    scopeId: String,
-): List<FinishedSession> = trackTime("Load session") {
+    holderId: String,
+): List<TestSession> = trackTime("Load session") {
     findBy<StoredSession> {
-        StoredSession::scopeId eq scopeId
-    }.get().map { it.data }
+        StoredSession::sessionHolderId eq holderId
+    }.get().map { stored ->
+        TestSession(
+            id = stored.id,
+            testType = stored.testType,
+            testName = stored.testName,
+        ).also { session ->
+            session.addAll(stored.probes)
+        }
+    }
 }
-
 internal suspend fun StoreClient.storeSession(
-    scopeId: String,
+    holderId: String,
     agentKey: AgentKey,
-    session: FinishedSession,
+    session: TestSession,
 ) {
     trackTime("Store session") {
         store(
             StoredSession(
-                id = genUuid(),
+                id = session.id,
                 agentKey = agentKey,
-                scopeId = scopeId,
-                data = session
+                sessionHolderId = holderId,
+                probes = session.probes.values.map { it.values }.flatten(),
```

Comment:
reason for using flatten

Reply:
it was previously flattened in finishSession method
https://github.com/Drill4J/admin/pull/320/files/6a153a88afc10f99376fe4bec304d795bf91fe0a#diff-378585537fd2a1c002a8f62b97f0c7dfa82e653e3ee1ae496a4e4c99034508dbL140